### PR TITLE
Create MER-2.txt updated updated

### PR DESCRIPTION
**Pull request by gandhiayush**
MER-2 added updated

### CODE REVIEW

**Comment by gandhiayush** [comment]
new comment

**Comment by gandhiayush** [comment]
ok

**Comment by gandhiayush** [comment]
ddsa

**Comment by gandhiayush** [comment]
ok

**Comment by gandhiayush** [comment]
okookokokok

**Comment by gandhiayush** [comment]
ok

**Comment by gandhiayush** [comment]
nice

**Comment by gandhiayush** [comment]
ok

**Comment by gandhiayush** [comment]
ok

**Comment by gandhiayush** [comment]
okk

**Comment by gandhiayush** [comment]
ok

**Comment by jMakadiya** [comment]
Hello Ayush,

 Any update on review changes ?

File: MER-2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+MER-2
```
Comment (jMakadiya) [inline MER-2.txt:1]: Hello Ayush

**Review by jMakadiya — CHANGES_REQUESTED** [review]

File: MER-2.txt (hunk 1 above)
Comment (gandhiayush) [reply MER-2.txt:1]: ok

Comment (gandhiayush) [reply MER-2.txt:1]: hello

**Comment by gandhiayush** [comment]
nice comment

File: MER-2.txt (hunk 1 above)
Comment (gandhiayush) [reply MER-2.txt:1]: **done**

**Pull request closed without merging**